evolver/device.py

```diff
@@ -28,6 +29,7 @@ class Config(BaseInterface.Config):
         interval: int = settings.DEFAULT_LOOP_INTERVAL
 
     def __init__(self, *args, **kwargs):
+        self.id = uuid.uuid4().hex[:6]
```

Comment:
There might be some scope for using this as a tracer amongst logs, history, & data, etc. Perhaps at least for sanity of which box produced what. In which case, this might be better off in the ``Config``.

Reply:
Just saw @amitschang slcak comment:
>Technically maybe the mac address of the underlying network device (e.g. on the rPi) might more stably identify the particular device?

I think this is a great idea as it gives users a way of physically knowing which box (hardware) produced stuff (as long as it also gets dumped out with the config and everything else).